src/blog/lagom-1.4.6.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Lagom 1.4.6 released!
+date: 2018-05-31
+author_github: ignasi35
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.4.6
+---
+
+The Lagom team has released Lagom 1.4.6.
+
+This release provides several bug fixes and updates.
+
+## Highlights:
+
+- Updates Akka to [2.5.13](https://akka.io/blog/news/2018/05/31/akka-2.5.13-released)
+- Updates Akka Persistence JDBC to [3.4.0](https://github.com/lagom/lagom/issues/1374)
+- Fix in testkit to [allow tests using Play's DBAPI](https://github.com/lagom/lagom/pull/1355)
+- Small fixes improving the error reporting for developers. E.g. improved error message in [some issues in the Service.Descriptor](https://github.com/lagom/lagom/issues/1362) and improved error message when there's [no connection settings for userland read-side DB](https://github.com/lagom/lagom/pull/1372)
+
+See GitHub for the [full list of commits](https://github.com/lagom/lagom/compare/1.4.5...1.4.6) and the issues and pull requests assigned to the [1.4.5 milestone](https://github.com/lagom/lagom/milestone/31?closed=1).
```

Comment:
Should say "1.4.6 milestone" (the link is correct).

Reply:
Thanks!